src/menu/menu.sss

```diff
@@ -68,6 +77,15 @@ a.menu_link
   .menu li &.chat
     height: 60px
 
+.menu li:last-child a::after
```

Comment:
What is a magic number here?

Reply:
Special selector for chat item

Reply:
It’s too hacky way. Can we avoiding putting exact size and use relative sizes like `calc(100% + 3px)`?

Reply:
calc() work in Menu but in Submenu it crashes it all because of overflow-x(y)…

Reply:
Can we just add `position: relative` to `li` and then use `top: -3px` at `li::after`?

We need to remove magic numbers since it will be hard to maintain them (easy to forget and easy to miss the mistake since you do not see `::after`)

Reply:
I understand the idea and i will try it little later